lib/generators/react_on_rails/setup_files_generator.rb

```diff
@@ -0,0 +1,26 @@
+require "rails/generators"
+
+module ReactOnRails
+  module Generators
+    class SetupFilesGenerator < Rails::Generators::Base
+      source_root File.expand_path("../templates", __FILE__)
+
+      def update_git_ignore
+        append_to_file ".gitignore" do
+          "# React on Rails\nnpm-debug.log\nnode_modules"
+        end
+      end
+
+      def update_application_js
+        insert_into_file "app/assets/javascripts/application.js",
+                         before: "//= require_tree ." do
```

Comment:
+1  I think deleting require_tree in application.js is a first two-minutes operation for many projects :)